src/renderer/pages/Profile/ProfileNew.vue

```diff
@@ -277,24 +242,12 @@ export default {
         return all
       }, {})
     },
-    networks () {
-      return this.$store.getters['network/all']
-    },
     defaultNetworks () {
-      return NETWORKS.map(network => {
-        return {
-          id: network.id,
-          title: network.description,
-          textContent: network.title,
-          svg: `networks/${network.id}.svg`
-        }
-      })
-    },
-    first () {
-      return this.defaultNetworks[0]
+      return NETWORKS.map(network => network)
```

Comment:
can't this simply be `return NETWORKS`?

Reply:
For some reason, the length appears empty when reloading the window.